testing/mod.ts

```diff
@@ -245,55 +245,181 @@ function green_ok() {
   return FG_GREEN + "ok" + RESET;
 }
 
-export async function runTests() {
-  let passed = 0;
-  let failed = 0;
-
-  console.log("running", tests.length, "tests");
-  for (let i = 0; i < tests.length; i++) {
-    const { fn, name } = tests[i];
-    let result = green_ok();
+export async function runIfMain(meta: ImportMeta, opts?: RunOptions) {
```

Comment:
Add jsdocs here explaining the defaults.

Reply:
updated